lib/netsuite/records/note.rb

```diff
@@ -0,0 +1,36 @@
+module NetSuite
+  module Records
+    class Note
+      OUTGOING = '_outgoing'
+      INCOMING = '_incoming'
```

Comment:
@charger what's the idea behind these constants? Is this something that the gem user will use?